### PR TITLE
ML Classifier Copies Tutorial

### PR DESCRIPTION
**Pull request by alberginia**
This PR corresponds to Milestone 10 of the PRESC ML Classifier Copies Package.

It incorporates a Tutorial which is a jupyter notebok with a practical part and detailed explanations on how to use the package. This is aimed at inexperienced users so they can understand and practice all the concepts and basic steps necessary to carry out the classifier copying process.

### CODE REVIEW

**Comment by alberginia** [comment]
The PR fails the checks but it doesn't look like it has anything to do with the changes introduced here (?).

**Comment by dzeber** [comment]
> The PR fails the CI checks but it doesn't look like it has anything to do with the changes introduced here (?).

Not sure why that happened, but I reran it and it passed.

**Review by dzeber — APPROVED** [review]
This is great! Please put a link to this in the ml_copies doc as well.

File: examples/ML_classifier_copies/ML_copies_tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,827 @@
...
+    "\n",
+    "Such an imbalance arises from a combination of the particular **topology of the class boundaries** learned by the original classifier and the **sampling strategy** to perform the copy. For instance, when using samplers that probe the space uniformly, the probability of having a class in the synthetic dataset is proportional to the volume of the space defined by its boundaries. Or when using a spherical sampler that assumes a gaussian distribution of the features, the probability of having a class in the synthetic dataset might increase for those classes that are located closer to the origin with respect to those in the outskirts, even if they occupy a similar volume.\n",
+    "\n",
+    "In order to tackle this problem, we can check for balance in the generated dataset, and then generate more samples if necessary until all classes have the same number of samples. In order to do this, we can simply set the parameter `enforce_balance` of the `ClassifierCopy` instance to `False`, or use a sampler that already does the balancing such as the spherical sampler.\n",
```
[the remark this answers is not in the document]
Comment (alberginia) [reply examples/ML_classifier_copies/ML_copies_tutorial.ipynb]: Yes, thanks! Fixed.

**Comment by alberginia** [comment]
> This is great! Please put a link to this in the ml_copies doc as well.

Ok! Done as well!  :)

**Pull request merged**